tech/languages/python/python-installation.md

```diff
@@ -3,31 +3,31 @@ title: Python
 subsection: python
 section: tech-languages
 order: 1
-version: 2.7.12
+version: 3.5.2
```

Comment:
I didn't see in the website where this information is used, but since the text talks about Python 3, seems reasonable to update.